src/styles/blocks/contact-us.scss

```diff
@@ -0,0 +1,115 @@
+.contact-us {
+  margin-bottom: 64px;
+
+  &__grid {
+    @include pageGrid;
+  }
+
+  &__form {
+    margin-bottom: 48px;
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+      margin-bottom: 80px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__contacts {
+    grid-column: span 2;
+
+    @include onTablet {
+      grid-column: span 6;
+      margin-bottom: 80px;
+    }
+
+    @include onDesktop {
+      grid-column: 8 / 13;
+      margin-bottom: 0;
+    }
+  }
+
+  &__input {
+    width: 100%;
+    height: 48px;
+    box-sizing: border-box;
+    padding: 14px 16px;
+
+    background: #f9f9f9;
```

Comment:
Consider using variables for colors like #f9f9f9 for consistency and maintainability.